server/bin/pbench-verify-backup-tarballs.py

```diff
@@ -120,10 +120,12 @@ def s3_entry_list_creation(self):
             while True:
```

Comment:
Can we use a variable here?  Infinite loops can be a bit nerve racking to look at at time.

Reply:
Sure.